frontend/libs/studio-components/src/components/StudioToggleableTextfield/StudioTextfieldToggleView/StudioTextfieldToggleView.module.css

```
@@ -1,9 +1,27 @@
+.button {
+  display: flex;
+  justify-content: flex-start;
```

If we do `space-between` here, do we need the `text-align: right' on `.editIconWrapper` then? 

```suggestion
  justify-content: space-between;
```